src/lib/ui/toast.ts

```diff
@@ -15,7 +15,7 @@ export function show_toast(baheth_link, type: "video" | "playlist") {
   // show the toast after 300ms of creation
   setTimeout(() => {
```

Comment:
Do we still need the timeout in this case?

Reply:
I'm not sure. I will check out.